installer/charts/rhtap-infrastructure/templates/odf/enable-odf-console-plugin.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.odf.enabled }}
+apiVersion: batch/v1
+kind: Job
```

Comment:
I've found that Jobs tend to cause issues. Can you switch to a Pod, and make sure it is idempotent?

Reply:
Hum... right now we are using more Jobs than plain pots for post-install hooks. Are we planning to clean that up? What problems have you seen? This is one of the reasons jobs exist. https://github.com/search?q=repo%3Aredhat-appstudio%2Frhtap-cli%20post-install&type=code. Also, they provide an example in helm docs with Job https://helm.sh/docs/topics/charts_hooks/

Reply:
Thanks for time in the call to clarify, just letting others know we decided to leave jobs for now with some limited backoffLimit to avoid long time to fail with 6 being the default